client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.js

```diff
@@ -160,41 +171,46 @@ const DataFilesProjectMembers = ({
         Add Member
       </Label>
       <div styleName="user-search">
-        <div className="input-group">
-          <div className="input-group-prepend">
-            <Button
-              styleName="add-button member-search"
-              onClick={() => onAdd({ user: selectedUser, access: 'edit' })}
-              disabled={
-                !selectedUser ||
-                loading ||
-                alreadyMember(selectedUser) ||
-                mode === 'transfer'
-              }
-            >
-              Add
-            </Button>
-          </div>
-          <Input
-            list="user-search-list"
-            type="text"
-            onChange={e => userSearch(e)}
-            placeholder="Search by name"
-            styleName="member-search"
-            disabled={loading || mode === 'transfer'}
-          />
-          <datalist id="user-search-list">
-            {/* eslint-disable */
+        <form ref={formRef}>
```

Comment:
console error here: `validateDOMNesting(...): <form> cannot appear as a descendant of <form>.`